components/base/wheeled/wheeled_base.go

```diff
@@ -338,14 +339,14 @@ func (base *wheeledBase) IsMoving(ctx context.Context) (bool, error) {
 	return false, nil
 }
 
-// Close is called from the client to close the instance of the base.
-func (base *wheeledBase) Close(ctx context.Context) error {
-	return base.Stop(ctx, nil)
+// Close is called from the client to close the instance of the wb.
```

Comment:
[nit] would rather the variable spelled out in the comments, i know f2 probably renamed them in vscod.